frontend/src/components/Footer.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import { css, StyleSheet } from 'aphrodite';
+
+export default class Footer extends Component {
+
+  render() {
+    return <div className={css(styles.footer)}>bears-21 2017<a href="https://www.github.com/chingu-voyage3/bears-21">Github</a>.</div>;
+  }
+
+}
+
+
+const styles = StyleSheet.create({
+  footer: {
+    bottom: 6,
```

Comment:
`6px`, `6em` ...?

Reply:
When used in aphrodite instead of writing `"6px"` you can short hand it to 6, saves you writing the `""`